### ISSUE TITLE
Unable to login after changing port 80

### ISSUE THREAD

**Issue by zahidhanif** [unmonoqueteclea/voilib#3]
I can’t use port 80 on my host. When I try and change the port in the compose file, I am unable to login. Can the port be changed?

**Comment by unmonoqueteclea** [comment]
Yes, I am working on this and I expect it to be ready during the following
days.

**Comment by unmonoqueteclea** [comment]
I've just pushed a fix for this. See `readme` file.
Environment variables can be used now to configure ports:

- `VOILIB_MANAGEMENT_PORT` (for management page: default 8501)
- `VOILIB_FRONTEND_PORT` (for frontend: default 80)
- `VOILIB_API_PORT` (for backend: default 81)

**Comment by zahidhanif** [comment]
Thanks, I have pulled the latest compose file and set the 3 environment variables:
VOILIB_MANAGEMENT_PORT=8501
VOILIB_FRONTEND_PORT=82
VOILIB_API_PORT=81
After issuing the compose up command, I get "Error starting userland proxy: listen tcp4 0.0.0.0:80: bind: address already in use"

**Comment by unmonoqueteclea** [comment]
I think there is something else going on with `userland`. I don't have any experience at all with it, so it's difficult to me to guess.

I tried to run the new compose file with those environment variables and frontend was correctly using the new port: `82`.

I will reopen this to check if there are users having the same problem...

**Comment by zahidhanif** [comment]
Just wondering if anyone else has reported this issue?

**Comment by unmonoqueteclea** [comment]
I tried this again, and the frontend runs without any problem in the port 82 instead of the default 80

```bash
unmonoqueteclea:~/Downloads/voilib$ export VOILIB_FRONTEND_PORT=82
unmonoqueteclea:~/Downloads/voilib$ docker compose up
```

Did you stopped old containers from previous versions of Voilib first?

This shouldn't return anything:

```
docker ps | grep "voilib"
```

**Comment by zahidhanif** [comment]
Thanks for the reply. I think I am going to give up on this as it's still not working for me.

I tried the following on a clean setup:

```
curl https://raw.githubusercontent.com/unmonoqueteclea/voilib/main/compose.yml -o docker-compose.yaml
export VOILIB_MANAGEMENT_PORT=8501
export VOILIB_FRONTEND_PORT=82
export VOILIB_API_PORT=81
sudo docker-compose up -d
```

I still get the error message:
`Error response from daemon: driver failed programming external connectivity on endpoint voilib-frontend-1 (3262676781e059802b0971a50f8bbfab943f1b399c8ae4d0e49fe36fedbc1e9e): Error starting userland proxy: listen tcp4 0.0.0.0:80: bind: address already in use`
